test/scene-integration.test.ts

```diff
@@ -560,11 +554,12 @@ describe('Scene', () => {
           const sprite = scene.createSprite();
           sprite.enter((s: SpriteView) => {
             s.BorderColorOpacity = .4;
+            s.BorderPlacement = 1;  // Place border outside shape.
```

Comment:
These changes ensure that the `hitTest()` succeeds only if the bounding box it tests includes an external border when specified.

Reply:
Done.